content/ja/monitors/guide/scoping_downtimes.md

```diff
@@ -47,7 +47,7 @@ title: ダウンタイムスケジュールのスコープ
 
 ## モニタータグで
 
-スケジュールされたダウンタイムが共通のモニタータグに基づいており、スコープ内のモニターが 1 つの group by スコープを持つマルチアラートモニターである場合、`Group scope` フィールドを使用して、スコープ内のモニターが共通に持つグループをサイレントにできます。 
+スケジュールされたダウンタイムが共通のモニタータグに基づいており、スコープ内のモニターが 1 つの group by スコープを持つマルチアラートモニターである場合、`Group scope` フィールドを使用して、スコープ内のモニターが共通に持つグループをサイレントにできます。
```

Comment:
@Dalje-et did you edit the Japanese docs as well? These are automated, for future reference.